src/formatDate.js

```diff
@@ -51,6 +51,48 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const dataHolder = {
+  };
+
+  let newData = '';
+
+  const lastIndex = fromFormat.length - 1;
+
+  const oldDateformats = fromFormat.slice(0, lastIndex);
+  const oldSeparator = fromFormat.slice(lastIndex);
+
+  const oldDateArray = date.split(oldSeparator);
+
+  // need to separate date to an array;
+  for (let i = 0; i < 3; i++) {
+    dataHolder[oldDateformats[i]] = oldDateArray[i];
+  }
+
+  // couldnt put getter in object, so i put it here;
+  // Oh well
+  if (!dataHolder.YY) {
```

Comment:
It's good to compare your value to something, instead of casting it to boolean. This way you know what is checked here.
```suggestion
  if (dataHolder.YY === undefined) {
```